src/Components/AsideMenu/Menu/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Accordion from './Accordion';
+import './index.scss';
+
+class Menu extends Component {
+  render() {
+    // TODO: 로그인 사용자 정보로 바꾸기
+
+    const { isOpen, user, characters, categories } = this.props;
+    return (
+      <div
+        className={`sideMenu ${isOpen && 'open'}`}
+        onClick={(e) => e.stopPropagation(e)}
+      >
+        <div className="menuContents">
+          <article className="userInfoWrap">
+            <p className="userHiTxt">
+              {user ? (
+                <>
+                  <Link to="/">{user}</Link>
+                  <span>님 반가워요!</span>
+                </>
+              ) : (
+                <>
+                  <Link to="/">로그인</Link>
+                  <span>이 필요해요!</span>
+                </>
+              )}
+            </p>
+            <Link to="/" className="alertBell"></Link>
+          </article>
+          <ul>
+            <li className="topPadding">
+              <Link to="/">장바구니 내역</Link>
+            </li>
+            <li className="borderPaddingBottom">
+              <Link to="/">주문·배송 내역</Link>
+            </li>
+            <li className="topPadding">
+              <Accordion
+                type={'character'}
+                title={'캐릭터'}
+                characters={characters}
+              />
```

Comment:
```suggestion
              <Accordion
                type='character'
                title='캐릭터'
                characters={characters}
              />
```
단순 문자열만 사용한다면 중괄호 불필요할 것 같아요

Reply:
수정했습니다.